Assets/Scripts/SS3D/Systems/Inventory/Items/ItemSystem.cs

```diff
@@ -93,7 +93,7 @@ public Item SpawnItem(ItemId id, Vector3 position, Quaternion rotation)
         /// <param name="id">The item ID to spawn.</param>
         /// <param name="container">The container to spawn into.</param>
         [Server]
-        public Item SpawnItemInContainer(ItemId id, Container container)
+        public Item SpawnItemInContainer(ItemId id, AttachedContainer container)
```

Comment:
renamed container to attachedContainer, having "container.container" in the code is confusing

Reply:
done